journal-of-investigative-dermatology.csl

```diff
@@ -1,227 +1,309 @@
+    <issnl>0022-202X</issnl>
+    <summary>This style based on the Vancouver author-date style</summary>
+    <updated>2016-09-23T20:19:12+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <locale xml:lang="en">
+    <date form="text" delimiter=" ">
+      <date-part name="year"/>
+      <date-part name="month" form="short" strip-periods="true"/>
+      <date-part name="day"/>
+    </date>
     <terms>
-      <term name="editor" form="short">
-        <single>ed.</single>
-        <multiple>eds</multiple>
-      </term>
+      <term name="retrieved">available</term>
+      <term name="section" form="short">sect.</term>
     </terms>
   </locale>
-  <macro name="container">
-    <choose>
-      <if type="chapter paper-conference" match="any">
-        <text term="in" form="long" plural="false" text-case="capitalize-first" font-style="italic" suffix=": "/>
-        <text variable="container-title" font-style="italic" form="short"/>
-        <text variable="collection-title" prefix=", "/>
-        <names variable="editor translator" prefix=" (" delimiter=", " suffix=")">
-          <name delimiter-precedes-last="never" initialize-with="" name-as-sort-order="all"/>
-          <label form="short" strip-periods="false" prefix=" "/>
-        </names>
-      </if>
-      <else>
-        <group delimiter=", ">
-          <text variable="container-title" form="short" strip-periods="true" font-style="italic"/>
-          <text variable="collection-title"/>
-        </group>
-      </else>
-    </choose>
-  </macro>
-  <macro name="author">
+  <locale xml:lang="fr">
+    <date form="text" delimiter=" ">
+      <date-part name="day"/>
+      <date-part name="month" form="short" strip-periods="true"/>
+      <date-part name="year"/>
+    </date>
+    <terms>
+      <term name="retrieved">disponible</term>
+      <term name="from">sur</term>
+    </terms>
+  </locale>
+  <locale xml:lang="de">
+    <terms>
+      <term name="retrieved">verfügbar</term>
+      <term name="from">unter</term>
+    </terms>
+  </locale>
+  <macro name="author-short-in-citation">
     <names variable="author">
-      <name delimiter-precedes-last="never" initialize-with="" name-as-sort-order="all"/>
-      <label form="short" strip-periods="false" prefix=" (" suffix=")"/>
-      <et-al font-style="italic"/>
+      <name form="short" and="text"/>
       <substitute>
         <names variable="editor"/>
         <names variable="translator"/>
-        <text macro="title"/>
       </substitute>
     </names>
   </macro>
-  <macro name="author-short">
+  <macro name="author-in-citation">
     <names variable="author">
-      <name form="short" and="text" initialize-with=". "/>
+      <name sort-separator=" " initialize-with="" name-as-sort-order="all" delimiter=", " delimiter-precedes-last="always"/>
```

Comment:
this almost certainly needs a `form="short"`. I'd also be surprised if you didn't want either `and="text"` or `and="symbol"`